CHANGELOG.md

```diff
@@ -118,6 +118,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - We fixed an issue where a database exception related to a missing timezone was too big. [#4827](https://github.com/JabRef/jabref/issues/4827)
 - We fixed an issue where the command line help text had several errors, and arguments and descriptions have been rewritten to simplify and detail them better. [#4932](https://github.com/JabRef/jabref/issues/2016)
 - We fixed an issue where the same menu for changing entry type had two different sizes and weights. [#4977](https://github.com/JabRef/jabref/issues/4977)
+- We fixed an issue where the background color of all dialogs in the dark theme was the same as in the light one. [#4987](https://github.com/JabRef/jabref/issues/4987)
```

Comment:
As the darkmode was not officially released as part of 4.3.1 you can omit the comment here.

Reply:
Sure! Thank you, @matthiasgeiger. Now it should be done.